README.md

```diff
@@ -1,50 +1,41 @@
+When developers first began defining containers to structure HTML, they had only one generic element available to them: the `div`. Creating complex page layouts then required dozens of `div` elements that were often difficult to organize or locate within the code. They needed a way to distinguish one `div` from another, which led to `id` and `class` attributes on elements being misused in an attempt to communicate what the `div` was doing. For example, a document usually has only one header, so it seemed sensible to write `div id="header"` as a way to say something stronger than, "this is a text division." Developers wanted to say, "this is a special block of introduction," but lacked the specific language to express it. They wanted those sections to have a semantic meaning.
 
 ```html
 <div id="header">
   <div class="wrapper">...</div>
 </div>
 ```
 
-When the W3C (the organization that oversees the specifications for HTML & CSS) started writing the specification for HTML5 they wanted to create new elements that would eliminate the need to label so many divs and make our code more readable for developers and more descriptive to browsers. Along with the help of Google, hundreds of thousands of websites were scanned and all the most common id and class names were recorded. It turned out that many developers were in fact using the same names to label their elements. Such as `id="header"` or `"footer"`, `"main"`, `"nav"`, `"section"`, `"article"`, `"aside"`, etc. For each of these most common labels we now have elements we can use instead. So instead of
+When the W3C (the organization that oversees the specifications for HTML and CSS) started writing the specification for HTML5 they wanted to create new elements that would eliminate the need to label so many `div` elements. The goals were to make the code more readable for developers and more descriptive for browsers. It turned out that many developers were already using the same names to label their elements, such as `id="header"`, `id="footer"`, `id="nav"`, `class="article"`, etc. So HTML5 provided semantic elements that explicitly described those functions for developers to use instead.
+
+We once used to have to identify a `div` as our header section.
 
 ```html
 <div id="header">...</div>
 ```
 
-we can use
+Now we use the `header` element.
 
 ```html
 <header></header>
 ```
 
-The one thing all semantic elements have in common is that we should use them when the content within the element has related meaning.
-
-> se·man·tic
-> səˈman(t)ik/
-> adjective
->    relating to meaning in language or logic.
+Why do we call these semantic elements? "Semantic" indicates a relationship of meaning. Semantic elements are elements that we use when the content within the element all has the same related meaning. In our `header` example above, all the content we would put within the `header` element would relate to introductory content, such as titles or navigation.
```

Comment:
"Semantic" indicates a relationship of meaning. is a bit unclear to me. Maybe cut that sentence out, and just have the explanatory sentence after it?

Reply:
Good call.